src/main/java/com/duckblade/osrs/toa/features/invocationpresets/InvocationPresetsManager.java

```diff
@@ -86,50 +90,16 @@ public void shutDown()
 	}
 
 	@Subscribe
-	public void onGameTick(GameTick e)
+	public void onScriptPostFired(ScriptPostFired event)
 	{
-		if (!raidStateTracker.isInLobby())
+		// This is run when the party screen is brought up, whenever a tab is changed, and whenever an invocation is
+		// clicked.
+		if (event.getScriptId() == SCRIPT_ID_BUILD_TOA_PARTY_INTERFACE)
```

Comment:
Just noticed that toggling the rewards interface will cause an invocation rebuild that isn't accounted for here. Firing this on script `6732` as well should fix that.